doc/releases.md

```diff
@@ -0,0 +1,57 @@
+# Releases
+
+Shields is a community project that is stewarded by a handful of core maintainers who contribute on a volunteer basis. We do our best to maintain the availability and reliability of the service, and enhance and improve the project overall. However, if you've spotted something wrong or would like to see a specific feature implemented, please consider helping us resolve it by submitting a pull request. All community contributions, even documentation improvements, are welcome!
+
+https://github.com/badges/shields is a monorepo and hosts the Shields frontend and server code as well as the [badge-maker][npm package] NPM library (and the [badge design specification](https://github.com/badges/shields/tree/master/spec)). The packaging and release processes for these items is described in the respective sections below.
+
+## badge-maker package
+
+We follow [Semantic Versioning](https://semver.org/) for the badge-maker package, and publish git Tags using the form `X.Y.Z`, and all such tags of this form are badge-maker releases (e.g. [3.3.0](https://github.com/badges/shields/releases/tag/3.3.0))
+
+The [badge-maker][npm package] is also published to the npm.js registry.
+
+Releases of badge-maker are done as and when needed, and not on any predetermined interval.
+
+## Shields.io service
+
+The [Shields.io Service][shields.io] consists of the frontend [https://shields.io][shields.io] website which allows users to browse and discover available badges, as well as the badge server backend that serves up requested badges (e.g. https://img.shields.io/badge/badges-rock-blue).
+
+This is the core, free, anonymous service available for anyone to use and which serves up more than 500 million badges every month!
+
+We do not have a fixed schedule for deploying updates to the Shields.io production environment, but we typically deploy the latest version at least once per week.
+
+We do not have any guaranteed SLA for the Shields.io service, but we do have monitoring and observability capabilities in place and our [Operations team](https://github.com/badges/shields#project-leaders) will review an address any availability, performance, etc. issues on a best-effort basis.
```

Comment:
Typo: `and address`

Reply:
yeah aside from this one minor copy edit I'm happy to give this the :+1: and I'll go back and have a quick look at the self-hosting docs in this light.

Reply:
oops, good catch! 5fdc25988cac774557c5795160fc28aaad497550